File: src/unl/cse/DNA.java
package unl.cse;

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class DNA {

	public static void main(String args[]) {

		String fileName = "data/H1N1nucleotide.txt";
		Scanner s = null;
		try {
			s = new Scanner(new File(fileName));
		} catch (FileNotFoundException e) {
			e.printStackTrace();
		}

		String dna = "";

		while(s.hasNext()) {
			dna += s.next().trim();
		}
		s.close();

		String subsequence = null;

		int count = 0; 
		//write code to count the number of times subsequence appears in the dna string

		System.out.println(subsequence + " appears " + count + " times");
		
	}
	
}

File: PHP/dna.php
<?php

  $file_handle = fopen("../data/H1N1nucleotide.txt", "r");

  $dna = "";
  while(!feof($file_handle)) {
    $line = fgets($file_handle);
    $dna .= trim($line);
  }
  fclose($file_handle);

  $subsequence = null;
  
  $count = 0;
  //TODO: write code to count the number of instances of the subsequence in the dna string



  print $subsequence . " appears " . $count . " times by direct inspection\n";

?>
